wordpress/wp-content/plugins/sil-dictionary-webonary/include/browseview_func.php

```diff
@@ -685,21 +684,21 @@ function reversalindex($display, $chosenLetter, $langcode, $reversalnr = "")
 		else
 		{
 			$display .=  "<div id=searchresult class=" . $background . " style=\"clear:both;\">";
-				$display .=  "<div id=englishcol>";
+			$display .=  "<div id=englishcol>";
 
-				if($reversal->English != $englishWord)
-				{
-					$display .=  $reversal->English;
-				}
-				$englishWord = $reversal->English;
-				 $display .=  "</div>";
+			if($reversal->English != $englishWord)
```

Comment:
The variable `$englishWord` is undefined.

Reply:
I think this was used to keep same head entries from being repeated in successive rows of a tabular display. So I fixed the logic and changed the variable name to $previousEnglishWord for clarity.